blob/blob.go

```diff
@@ -41,6 +41,7 @@ type Client interface {
 	DeleteAll(ctx context.Context, userID string) error
 	Get(ctx context.Context, id string) (*Blob, error)
 	GetContent(ctx context.Context, id string) (*Content, error)
+	GetDeviceLogsContents(ctx context.Context, userID string, filter *DeviceLogsFilter, pagination *page.Pagination) ([]*DeviceLogsContentRaw, error)
```

Comment:
I'm not sure this design is correct. For consistency with `GetContent`, we should be able to fetch contents of a single entry given an `id`, instead of returning the contents of multiple blobs.

Reply:
True but @nhamming requested yesterday a way to get the entire range of contents.